README.md

```diff
@@ -998,73 +1554,129 @@ The user of the jsvc process when `use_init => true`
+
+Default value: If you don't specify an `app_base`, Puppet deploys the WAR to your specified `deployment_path`. If you don't specify that either, the WAR deploys to `${catalina_base}/webapps`.
 
 #####`catalina_base`
 
-Specifies the base directory of the Tomcat installation. Valid options: a string containing an absolute path. Default: $::tomcat::catalina_home.
+Specifies the base directory of the Tomcat installation.
+
+Valid options: a string containing an absolute path.
+
+Default value: `$::tomcat::catalina_home`.
 
 #####`deployment_path`
 
-Specifies where to deploy the WAR. Cannot be used in combination with `app_base`. Valid options: a string containing an absolute path. Default: If you don't specify a `deployment_path`, Puppet deploys the WAR to your specified `app_base`. If you don't specify that either, the WAR deploys to '${catalina_base}/webapps'.
+Specifies where to deploy the WAR. Cannot be used in combination with `app_base`.
+
+Valid options: a string containing an absolute path.
+
+Default value: If you don't specify a `deployment_path`, Puppet deploys the WAR to your specified `app_base`. If you don't specify that either, the WAR deploys to `${catalina_base}/webapps`.
 
 #####`war_ensure`
 
-Specifies whether the WAR should exist. Valid options: 'true', 'false', 'present', and 'absent'. Default: 'present'.
+Specifies whether the WAR should exist.
+
+Valid options: `true`, `false`, `present`, and `absent`.
+
+Default value: `present`.
 
 #####`war_name`
 
-Specifies the name of the WAR. Valid options: a string containing a filename that ends in '.war'. Default: the '[name]' passed in your define.
+Specifies the name of the WAR.
+
+Valid options: a string containing a filename that ends in '.war'.
+
+Default value: the `name` passed in your define.
 
 #####`war_purge`
 
-Specifies whether to purge the exploded WAR directory. Only applicable when `war_ensure` is set to 'absent' or 'false'.
+Specifies whether to purge the exploded WAR directory. Only applicable when `war_ensure` is set to `absent` or `false`.
 
-**Note:** Setting this parameter to 'false' does not prevent Tomcat from removing the exploded WAR directory if Tomcat is running and autoDeploy is set to 'true'. Valid options: 'true' and 'false'. Default: 'true'.
+Valid options: `true` and `false`.
+
+Default value: `true`.
+
+>Note: Setting this parameter to `false` does not prevent Tomcat from removing the exploded WAR directory if Tomcat is running and autoDeploy is set to `true`.
 
 #####`war_source`
 
-*Required unless `war_ensure` is set to 'false' or 'absent'.* Specifies the source to deploy the WAR from. Valid options: a string containing a `puppet://`, `http(s)://`, or `ftp://` URL.
+*Required unless `war_ensure` is set to `false` or `absent`.* Specifies the source to deploy the WAR from.
+
+Valid options: a string containing a `puppet://`, `http(s)://`, or `ftp://` URL.
 
 ##Limitations
 
```

Comment:
Hmm, that's tricky because it's not a code or string. Really, it's jargon, so let's replace it with Unix-like systems. (I'm guilty of using this, so I'm glad you mentioned it @pmcmaw !)